File: keychain-sample-chat-iphone/KeychainChat/Models/Chat/ChatMessage.swift
//
//  ChatMessage.swift
//  KeychainChat
//
//  Created by Robert Ellis on 2022/05/31.
//

import Foundation

struct ChatMessage: Codable, Identifiable, Hashable {
    var id: String?
        
    var chatId: String?
    
    var sendOrRcvd: ChatDirection?

    var senderId: String?

    var receiverId: String?

    var imageUrl: String?
    
    var msg: String?
    
    var timestamp: Int64?
}

File: keychain-sample-chat-android/app/src/main/java/io/keychain/chat/models/chat/ChatMessage.java
package io.keychain.chat.models.chat;

public class ChatMessage {
    public String id;

    public String chatId;

    public ChatDirection sendOrRcvd;

    public String senderId;

    public String receiverId;

    public String imageUrl;

    public String msg;

    public Long timestamp;
}
